google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -224,27 +226,29 @@ def copy_file source_bucket_name, source_file_path,
         def rewrite_file source_bucket_name, source_file_path,
```

Comment:
The other difference is in how the encryption key is handled. `copy_file` take just `key`, but rewrite_file takes `source_key` and `destination_key`.